src/hub.html

```diff
@@ -33,7 +33,7 @@
     <a-scene
         renderer="antialias: true"
         networked-scene="adapter: janus; audio: true; debug: true; connectOnLoad: false;"
-        physics="gravity: -6;"
+        physics="gravity: -6; debug: true;"
```

Comment:
debug

Reply:
Yep, have that in the todo items to remove, its left on to help test during review